Aptoide/aptoide-client-v8
# Feature/asv 1947 new grid view for more fragment apps

**What does this PR do?**

This PR implements the new gridLayout logic for the more fragments in home. This should work for every more press done from home bundle that displays an app grid (ads included);

**Database changed?**

No

**Where should the reviewer start?**

- [ ] ListAppsMoreFragment.java

**How should this be manually tested?**

Check out the more on home bundles. Make sure the behaviour is kept (pull to refresh, load more, etc) and that the grid adapts as intended to the screen;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1947](https://aptoide.atlassian.net/browse/ASV-1947)

**Code Review Checklist**
